src/styles/blocks/moyo-header.scss

```diff
@@ -0,0 +1,71 @@
+html {
+  font-family: "Roboto", "Montserrat", sans-serif;
+  font-size: 12px;
+}
```

Comment:
You don't need to duplicate this code. Just set what you need one time in main.scss file